.github/workflows/ci.yml

```diff
@@ -27,6 +27,9 @@ jobs:
         run: uv run ruff format --check
       - name: Check lockfile
         run: uv lock --locked
+      - name: Pyrefly Python code
+        run: uv run pyrefly check
+        continue-on-error: true
```

Comment:
Is the GitHub annotation output the default? I noticed it was here before, but you remove it then.

Reply:
GitHub format is not supported by Pyrefly, only text and JSON is.